Sources/Leaf/Buffer/Buffer+Leaf.swift

```diff
@@ -126,12 +126,24 @@ extension Buffer {
     mutating func extractInstruction(stem: Stem) throws -> TagTemplate {
         let name = try extractInstructionName()
         let parameters = try extractInstructionParameters()
-
-        // check if body
-        moveForward()
-        guard current == .space, next == .leftCurlyBracket else {
+        
+        // check if body exists
+        
+        // Check for immediate curly brace without whitespace.
+        // If there is an immediate brace, do nothing.
+        if next != .leftCurlyBracket {
+            moveForward()
+        }
+        
+        // Move through any redundant whitespace
+        while current?.isWhitespace == true && next?.isWhitespace == true {
```

Comment:
Hi Logan, Thanks, I didn't see the method for this. It seems to be used in the buffer class in `ParameterParser` but using some functions from `StaticDataBuffer`, and will need a pretty solid refactor since it's not available. There is quite a lot of function calls going on in there - while I was trying to understand it right just now you merged the PR, so I think the simple conditional is ok?